build/js/src/triniti/sys/RedirectId.js

```diff
@@ -0,0 +1,51 @@
+import { Base64 } from 'js-base64';
```

Comment:
@hunyoboy use the already included/available lib `base-64`... see `AbstractBinaryType` in js lib.